sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>nit: Deploy the model to a Sagemake --&gt; Deploy the model to a Sagemaker</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed